src/Form/Type/CustomUrlType.php

```diff
@@ -40,7 +63,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('url', TextType::class, [
                 'label' => $this->trans('URL', 'Modules.Linklist.Admin'),
                 'required' => true,
-                'constraints' => [new Assert\Url()],
+                // Regex adapted from src/Symfony/Component/Validator/Constraints/UrlValidator.php (symfony 3.4)
+                // In symfony 4.4 this would be as simple as `new Assert\Url(['relativeProtocol' => true])`
+                'constraints' => [new Assert\Regex(['pattern' => $pattern])],
```

Comment:
```suggestion
 // use PrestaShop\PrestaShop\Core\Foundation\Version;
                'constraints' => [
                    Version::buildFromString(_PS_VERSION_)->isGreaterThan('8.0') 
                    ? new Assert\Url(['relativeProtocol' => true])
                    : new Assert\Regex(['pattern' => $pattern])
                ],
```
Maybe you can do something like that if the PrestaShop version is greater than 8 or simpler:

```suggestion
 // use Symfony\Component\HttpKernel\Kernel;
                'constraints' => [
                    version_compare (Kernel::VERSION, '4.4') >= 0
                    ? new Assert\Url(['relativeProtocol' => true])
                    : new Assert\Regex(['pattern' => $pattern])
                ],
```